src/main/scala/scoverage/ScoverageSbtPlugin.scala

```diff
@@ -56,11 +56,12 @@ object ScoverageSbtPlugin extends AutoPlugin {
     addCommandAlias("coverageOn", ";set ThisBuild / coverageEnabled := true") ++
     addCommandAlias("coverageOff", ";set ThisBuild / coverageEnabled := false")
 
-  override def projectSettings: Seq[Setting[_]] = Seq(
+  override def projectSettings: Seq[Setting[_]] = super.projectSettings ++ Seq(
```

Comment:
So I get that this is getting the current `projectSettings` and then appending, but I don't often seen this done is sbt plugins. What is the main reason for this change?

Reply:
Now that I think about it I don't think it's necessary and I will remove it.